src/mon/AuthMonitor.cc

```diff
@@ -1810,44 +1810,19 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
       goto done;
     }
 
-    KeyServerData::Incremental auth_inc;
```

Comment:
nit: typo in the commit title (`guthMonitor`).

Reply:
Fixed.